### PR TITLE
TS-13560 Update Maven and Eclipse configuration for dependencies mana…

### PR DESCRIPTION
**Pull request by MiguelUrrutiaSoler**
…gement

The tycho plugin configuration was updated and also eclipse project config files to work in the last eclipse version using one project conf for maven and pde development

### CODE REVIEW

File: plugins/com.contrastsecurity.ide.rest.sdk/.classpath [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-	<classpathentry exported="true" kind="lib" path="lib/contrast-sdk-java-3.3.2.jar" sourcepath="lib/contrast-sdk-java-3.3.2-sources.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/lombok-1.16.8.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
+	<classpathentry exported="true" kind="lib" path="lib/lombok-1.16.8.jar"/>
+	<classpathentry kind="src" path="src/"/>
+	<classpathentry exported="true" kind="lib" path="lib/contrast-sdk-java-3.3.2.jar" sourcepath="lib/contrast-sdk-java-3.3.2-sources.jar"/>
```
Comment (cgperez) [inline plugins/com.contrastsecurity.ide.rest.sdk/.classpath:7]: if java sdk is added through maven pom, why is this line required?

**Review by cgperez — CHANGES_REQUESTED** [review]

File: plugins/com.contrastsecurity.ide.rest.sdk/.classpath (hunk 1 above)
Comment (MiguelUrrutiaSoler) [reply plugins/com.contrastsecurity.ide.rest.sdk/.classpath:7]: The maven plug in updates the lib directory then the tycho plugin connector updates the config files needed in the PDE project that is the reason that this values appears in the classpath file

**Review by cgperez — APPROVED** [review]

**Review by bryanateoan — APPROVED** [review]
